spec/graphql/resolvers/customer_resolver_spec.rb

```diff
@@ -103,6 +109,30 @@
       expect(customer_response['hasCreditNotes']).to be true
       expect(customer_response['creditNotesCreditsAvailableCount']).to eq(1)
       expect(customer_response['creditNotesBalanceAmountCents']).to eq('120')
+      expect(customer_response['hasOverwrittenInvoiceCustomSectionsSelection']).to be true
+      expect(customer_response['skipInvoiceCustomSections']).to be false
+      expect(customer_response['applicableInvoiceCustomSections'].count).to eq(2)
+    end
+  end
+
+  context 'when customer has invoice_custom_sections nested from organization ' do
```

Comment:
not sure if nested is the right term here... isn't it inherited, assigned, applicable or something similar?